src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenRefReturnTests.vb

```diff
@@ -88,7 +88,7 @@ End Module",
   IL_0012:  stloc.0
   IL_0013:  ldloc.0
   IL_0014:  ldc.i4.2
-  IL_0015:  call       "Function C.F(Of Integer)(Integer(), Integer) As Integer"
+  IL_0015:  call       "Function C.F(Of Integer)(Integer(), Integer) ByRef As Integer"
```

Comment:
Yes, I think we settled on this form of display. I am going to address this in a separate change.